x/wasm/internal/types/types.go

```diff
@@ -203,14 +207,18 @@ func ParseEvents(logs []wasmTypes.EventAttribute, contractAddr sdk.AccAddress) s
 
 // WasmConfig is the extra config required for wasm
 type WasmConfig struct {
-	SmartQueryGasLimit uint64 `mapstructure:"query_gas_limit"`
-	CacheSize          uint64 `mapstructure:"lru_size"`
+	SmartQueryGasLimit uint64
```

Comment:
Looking to see where this is read/parsed, but surprised there are no explicit field naming here.

Reply:
The .toml file is parsed with a viper instance that had the wasm flags registered. We are not parsing the file in our app package anymore